### ISSUE TITLE
Error on date parsing

### ISSUE THREAD

**Issue by giampaolotrapasso** [DanielaSfregola/twitter4s-demo#1]
Hi Daniela,

I just forked this repo to get a try on your examples. Using example 4 I got this error:

```
[info] Running UserTopHashtags 
[INFO] [12/13/2015 21:42:49.363] [run-main-0] [ActorSystem(twitter4s)] GET https://api.twitter.com/1.1/statuses/user_timeline.json?contributor_details=false&count=200&exclude_replies=false&include_rts=true&screen_name=odersky&trim_user=false
[INFO] [12/13/2015 21:42:52.860] [twitter4s-akka.actor.default-dispatcher-2] [ActorSystem(twitter4s)] GET https://api.twitter.com/1.1/statuses/user_timeline.json?contributor_details=false&count=200&exclude_replies=false&include_rts=true&screen_name=odersky&trim_user=false (200 OK) | 3724ms
[error] (run-main-0) spray.httpx.PipelineException: No usable value for created_at
[error] Invalid date 'Mon Nov 23 11:54:21 +0000 2015'
spray.httpx.PipelineException: No usable value for created_at
Invalid date 'Mon Nov 23 11:54:21 +0000 2015'
    at spray.httpx.ResponseTransformation$$anonfun$unmarshal$1.apply(ResponseTransformation.scala:36)
    at spray.httpx.ResponseTransformation$$anonfun$unmarshal$1.apply(ResponseTransformation.scala:31)
    at spray.httpx.TransformerPipelineSupport$WithTransformation.$tilde$greater(TransformerPipelineSupport.scala:33)
    at com.danielasfregola.twitter4s.http.clients.OAuthClient$$anonfun$unmarshalResponse$1.apply(OAuthClient.scala:30)
    at com.danielasfregola.twitter4s.http.clients.OAuthClient$$anonfun$unmarshalResponse$1.apply(OAuthClient.scala:28)
    at scala.util.Success$$anonfun$map$1.apply(Try.scala:237)
    at scala.util.Try$.apply(Try.scala:192)
    at scala.util.Success.map(Try.scala:237)
    at scala.concurrent.Future$$anonfun$map$1.apply(Future.scala:235)
    at scala.concurrent.Future$$anonfun$map$1.apply(Future.scala:235)
    at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32)
    at akka.dispatch.BatchingExecutor$Batch$$anonfun$run$1.processBatch$1(BatchingExecutor.scala:67)
    at akka.dispatch.BatchingExecutor$Batch$$anonfun$run$1.apply$mcV$sp(BatchingExecutor.scala:82)
    at akka.dispatch.BatchingExecutor$Batch$$anonfun$run$1.apply(BatchingExecutor.scala:59)
    at akka.dispatch.BatchingExecutor$Batch$$anonfun$run$1.apply(BatchingExecutor.scala:59)
    at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:72)
    at akka.dispatch.BatchingExecutor$Batch.run(BatchingExecutor.scala:58)
    at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:41)
    at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(AbstractDispatcher.scala:393)
    at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
    at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
    at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
    at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
Caused by: org.json4s.package$MappingException: No usable value for created_at
Invalid date 'Mon Nov 23 11:54:21 +0000 2015'
    at org.json4s.reflect.package$.fail(package.scala:96)
    at org.json4s.Extraction$ClassInstanceBuilder.org$json4s$Extraction$ClassInstanceBuilder$$buildCtorArg(Extraction.scala:462)
    at org.json4s.Extraction$ClassInstanceBuilder$$anonfun$14.apply(Extraction.scala:482)
    at org.json4s.Extraction$ClassInstanceBuilder$$anonfun$14.apply(Extraction.scala:482)
    at scala.collection.TraversableLike$$anonfun$map$1.apply(TraversableLike.scala:245)
    at scala.collection.TraversableLike$$anonfun$map$1.apply(TraversableLike.scala:245)
    at scala.collection.mutable.ResizableArray$class.foreach(ResizableArray.scala:59)
    at scala.collection.mutable.ArrayBuffer.foreach(ArrayBuffer.scala:48)
    at scala.collection.TraversableLike$class.map(TraversableLike.scala:245)
    at scala.collection.AbstractTraversable.map(Traversable.scala:104)
    at org.json4s.Extraction$ClassInstanceBuilder.org$json4s$Extraction$ClassInstanceBuilder$$instantiate(Extraction.scala:470)
    at org.json4s.Extraction$ClassInstanceBuilder$$anonfun$result$6.apply(Extraction.scala:515)
    at org.json4s.Extraction$ClassInstanceBuilder$$anonfun$result$6.apply(Extraction.scala:512)
    at org.json4s.Extraction$.org$json4s$Extraction$$customOrElse(Extraction.scala:524)
    at org.json4s.Extraction$ClassInstanceBuilder.result(Extraction.scala:512)
    at org.json4s.Extraction$.extract(Extraction.scala:351)
    at org.json4s.Extraction$CollectionBuilder$$anonfun$6.apply(Extraction.scala:360)
    at org.json4s.Extraction$CollectionBuilder$$anonfun$6.apply(Extraction.scala:360)
    at scala.collection.immutable.List.map(List.scala:273)
    at org.json4s.Extraction$CollectionBuilder.mkCollection(Extraction.scala:360)
    at org.json4s.Extraction$CollectionBuilder.result(Extraction.scala:384)
    at org.json4s.Extraction$.extract(Extraction.scala:339)
    at org.json4s.Extraction$.extract(Extraction.scala:42)
    at org.json4s.ExtractableJsonAstNode.extract(ExtractableJsonAstNode.scala:21)
    at org.json4s.native.Serialization$.read(Serialization.scala:71)
    at com.danielasfregola.twitter4s.http.unmarshalling.JsonSupport$$anonfun$json4sUnmarshaller$1.applyOrElse(JsonSupport.scala:25)
    at com.danielasfregola.twitter4s.http.unmarshalling.JsonSupport$$anonfun$json4sUnmarshaller$1.applyOrElse(JsonSupport.scala:23)
    at scala.runtime.AbstractPartialFunction.apply(AbstractPartialFunction.scala:36)
    at spray.httpx.unmarshalling.Unmarshaller$$anon$1$$anonfun$unmarshal$1.apply(Unmarshaller.scala:29)
    at spray.httpx.unmarshalling.SimpleUnmarshaller.protect(SimpleUnmarshaller.scala:40)
    at spray.httpx.unmarshalling.Unmarshaller$$anon$1.unmarshal(Unmarshaller.scala:29)
    at spray.httpx.unmarshalling.SimpleUnmarshaller.apply(SimpleUnmarshaller.scala:29)
    at spray.httpx.unmarshalling.SimpleUnmarshaller.apply(SimpleUnmarshaller.scala:23)
    at spray.httpx.unmarshalling.UnmarshallerLifting$$anon$3.apply(UnmarshallerLifting.scala:35)
    at spray.httpx.unmarshalling.UnmarshallerLifting$$anon$3.apply(UnmarshallerLifting.scala:34)
    at spray.httpx.unmarshalling.UnmarshallerLifting$$anon$2.apply(UnmarshallerLifting.scala:30)
    at spray.httpx.unmarshalling.UnmarshallerLifting$$anon$2.apply(UnmarshallerLifting.scala:29)
    at spray.httpx.unmarshalling.package$PimpedHttpResponse.as(package.scala:51)
    at spray.httpx.ResponseTransformation$$anonfun$unmarshal$1.apply(ResponseTransformation.scala:33)
    at spray.httpx.ResponseTransformation$$anonfun$unmarshal$1.apply(ResponseTransformation.scala:31)
    at spray.httpx.TransformerPipelineSupport$WithTransformation.$tilde$greater(TransformerPipelineSupport.scala:33)
    at com.danielasfregola.twitter4s.http.clients.OAuthClient$$anonfun$unmarshalResponse$1.apply(OAuthClient.scala:30)
    at com.danielasfregola.twitter4s.http.clients.OAuthClient$$anonfun$unmarshalResponse$1.apply(OAuthClient.scala:28)
    at scala.util.Success$$anonfun$map$1.apply(Try.scala:237)
    at scala.util.Try$.apply(Try.scala:192)
    at scala.util.Success.map(Try.scala:237)
    at scala.concurrent.Future$$anonfun$map$1.apply(Future.scala:235)
    at scala.concurrent.Future$$anonfun$map$1.apply(Future.scala:235)
    at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32)
    at akka.dispatch.BatchingExecutor$Batch$$anonfun$run$1.processBatch$1(BatchingExecutor.scala:67)
    at akka.dispatch.BatchingExecutor$Batch$$anonfun$run$1.apply$mcV$sp(BatchingExecutor.scala:82)
    at akka.dispatch.BatchingExecutor$Batch$$anonfun$run$1.apply(BatchingExecutor.scala:59)
    at akka.dispatch.BatchingExecutor$Batch$$anonfun$run$1.apply(BatchingExecutor.scala:59)
    at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:72)
    at akka.dispatch.BatchingExecutor$Batch.run(BatchingExecutor.scala:58)
    at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:41)
    at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(AbstractDispatcher.scala:393)
    at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
    at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
    at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
    at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
Caused by: org.json4s.package$MappingException: Invalid date 'Mon Nov 23 11:54:21 +0000 2015'
    at org.json4s.Extraction$.formatDate(Extraction.scala:617)
    at org.json4s.Extraction$.convert(Extraction.scala:594)
    at org.json4s.Extraction$.extract(Extraction.scala:350)
    at org.json4s.Extraction$ClassInstanceBuilder.org$json4s$Extraction$ClassInstanceBuilder$$buildCtorArg(Extraction.scala:450)
    at org.json4s.Extraction$ClassInstanceBuilder$$anonfun$14.apply(Extraction.scala:482)
    at org.json4s.Extraction$ClassInstanceBuilder$$anonfun$14.apply(Extraction.scala:482)
    at scala.collection.TraversableLike$$anonfun$map$1.apply(TraversableLike.scala:245)
    at scala.collection.TraversableLike$$anonfun$map$1.apply(TraversableLike.scala:245)
    at scala.collection.mutable.ResizableArray$class.foreach(ResizableArray.scala:59)
    at scala.collection.mutable.ArrayBuffer.foreach(ArrayBuffer.scala:48)
    at scala.collection.TraversableLike$class.map(TraversableLike.scala:245)
    at scala.collection.AbstractTraversable.map(Traversable.scala:104)
    at org.json4s.Extraction$ClassInstanceBuilder.org$json4s$Extraction$ClassInstanceBuilder$$instantiate(Extraction.scala:470)
    at org.json4s.Extraction$ClassInstanceBuilder$$anonfun$result$6.apply(Extraction.scala:515)
    at org.json4s.Extraction$ClassInstanceBuilder$$anonfun$result$6.apply(Extraction.scala:512)
    at org.json4s.Extraction$.org$json4s$Extraction$$customOrElse(Extraction.scala:524)
    at org.json4s.Extraction$ClassInstanceBuilder.result(Extraction.scala:512)
    at org.json4s.Extraction$.extract(Extraction.scala:351)
    at org.json4s.Extraction$CollectionBuilder$$anonfun$6.apply(Extraction.scala:360)
    at org.json4s.Extraction$CollectionBuilder$$anonfun$6.apply(Extraction.scala:360)
    at scala.collection.immutable.List.map(List.scala:273)
    at org.json4s.Extraction$CollectionBuilder.mkCollection(Extraction.scala:360)
    at org.json4s.Extraction$CollectionBuilder.result(Extraction.scala:384)
    at org.json4s.Extraction$.extract(Extraction.scala:339)
    at org.json4s.Extraction$.extract(Extraction.scala:42)
    at org.json4s.ExtractableJsonAstNode.extract(ExtractableJsonAstNode.scala:21)
    at org.json4s.native.Serialization$.read(Serialization.scala:71)
    at com.danielasfregola.twitter4s.http.unmarshalling.JsonSupport$$anonfun$json4sUnmarshaller$1.applyOrElse(JsonSupport.scala:25)
    at com.danielasfregola.twitter4s.http.unmarshalling.JsonSupport$$anonfun$json4sUnmarshaller$1.applyOrElse(JsonSupport.scala:23)
    at scala.runtime.AbstractPartialFunction.apply(AbstractPartialFunction.scala:36)
    at spray.httpx.unmarshalling.Unmarshaller$$anon$1$$anonfun$unmarshal$1.apply(Unmarshaller.scala:29)
    at spray.httpx.unmarshalling.SimpleUnmarshaller.protect(SimpleUnmarshaller.scala:40)
    at spray.httpx.unmarshalling.Unmarshaller$$anon$1.unmarshal(Unmarshaller.scala:29)
    at spray.httpx.unmarshalling.SimpleUnmarshaller.apply(SimpleUnmarshaller.scala:29)
    at spray.httpx.unmarshalling.SimpleUnmarshaller.apply(SimpleUnmarshaller.scala:23)
    at spray.httpx.unmarshalling.UnmarshallerLifting$$anon$3.apply(UnmarshallerLifting.scala:35)
    at spray.httpx.unmarshalling.UnmarshallerLifting$$anon$3.apply(UnmarshallerLifting.scala:34)
    at spray.httpx.unmarshalling.UnmarshallerLifting$$anon$2.apply(UnmarshallerLifting.scala:30)
    at spray.httpx.unmarshalling.UnmarshallerLifting$$anon$2.apply(UnmarshallerLifting.scala:29)
    at spray.httpx.unmarshalling.package$PimpedHttpResponse.as(package.scala:51)
    at spray.httpx.ResponseTransformation$$anonfun$unmarshal$1.apply(ResponseTransformation.scala:33)
    at spray.httpx.ResponseTransformation$$anonfun$unmarshal$1.apply(ResponseTransformation.scala:31)
    at spray.httpx.TransformerPipelineSupport$WithTransformation.$tilde$greater(TransformerPipelineSupport.scala:33)
    at com.danielasfregola.twitter4s.http.clients.OAuthClient$$anonfun$unmarshalResponse$1.apply(OAuthClient.scala:30)
    at com.danielasfregola.twitter4s.http.clients.OAuthClient$$anonfun$unmarshalResponse$1.apply(OAuthClient.scala:28)
    at scala.util.Success$$anonfun$map$1.apply(Try.scala:237)
    at scala.util.Try$.apply(Try.scala:192)
    at scala.util.Success.map(Try.scala:237)
    at scala.concurrent.Future$$anonfun$map$1.apply(Future.scala:235)
    at scala.concurrent.Future$$anonfun$map$1.apply(Future.scala:235)
    at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32)
    at akka.dispatch.BatchingExecutor$Batch$$anonfun$run$1.processBatch$1(BatchingExecutor.scala:67)
    at akka.dispatch.BatchingExecutor$Batch$$anonfun$run$1.apply$mcV$sp(BatchingExecutor.scala:82)
    at akka.dispatch.BatchingExecutor$Batch$$anonfun$run$1.apply(BatchingExecutor.scala:59)
    at akka.dispatch.BatchingExecutor$Batch$$anonfun$run$1.apply(BatchingExecutor.scala:59)
    at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:72)
    at akka.dispatch.BatchingExecutor$Batch.run(BatchingExecutor.scala:58)
    at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:41)
    at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(AbstractDispatcher.scala:393)
    at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
    at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
    at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
    at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
[trace] Stack trace suppressed: run last compile:run for the full output.
```

I'm not using twitter4s from the source, but from maven.

**Comment by giampaolotrapasso** [comment]
Of course, I added an Await.result

**Comment by DanielaSfregola** [comment]
Hi Giampaolo, thank you for raising this issue!
Can I please know your JDK version? Also, can I have your pom?

I'll try to replicate the issue and fix it asap :smile:

**Comment by giampaolotrapasso** [comment]
For sure!
**pom.xml**

```
<?xml version='1.0' encoding='UTF-8'?>
<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0">
    <modelVersion>4.0.0</modelVersion>
    <groupId>default</groupId>
    <artifactId>twitter4s-demo_2.11</artifactId>
    <packaging>jar</packaging>
    <description>twitter4s-demo</description>
    <version>0.1</version>
    <name>twitter4s-demo</name>
    <organization>
        <name>default</name>
    </organization>
    <dependencies>
        <dependency>
            <groupId>org.scala-lang</groupId>
            <artifactId>scala-library</artifactId>
            <version>2.11.7</version>
        </dependency>
        <dependency>
            <groupId>com.danielasfregola</groupId>
            <artifactId>twitter4s_2.11</artifactId>
            <version>0.1</version>
        </dependency>
    </dependencies>
    <repositories>
        <repository>
            <id>Mavencentral</id>
            <name>Maven central</name>
            <url>http://repo1.maven.org/maven2/</url>
            <layout>default</layout>
        </repository>
    </repositories>
```

**java -version**

```
java version "1.8.0_05"
Java(TM) SE Runtime Environment (build 1.8.0_05-b13)
Java HotSpot(TM) 64-Bit Server VM (build 25.5-b02, mixed mode)
```

**Comment by DanielaSfregola** [comment]
Hi Giampaolo, this should now have been fixed in version 0.2.
